apps/prairielearn/src/lib/assets.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-check
```

Comment:
This is already a TypeScript file; why was this added?

Reply:
oops. fixed.